tests/Handler/CurlMultiHandlerTest.php

```diff
@@ -9,6 +9,31 @@
 
 class CurlMultiHandlerTest extends TestCase
 {
+    public function setUp()
+    {
+        $_SERVER['curl_test'] = true;
+        unset($_SERVER['_curl_multi']);
+    }
+
+    public function tearDown()
+    {
+        unset($_SERVER['_curl_multi'], $_SERVER['curl_test']);
+    }
+
+    public function testCanAddCustomCurlOptions()
```

Comment:
I see 2 options for solving the test failure here.

1. Use CURLMOPT_MAXCONNECTS, which existed from PHP 5.5.
1. Put a version guard on this test, something like:
```
/**
 * @requires PHP >= 7.0.7
 */
```

Reply:
Yes, I'm planning to put a version guard, because it's normal that some constants are available only in newer PHP versions.

Reply:
OK, replaced the constants with different ones (available in 5.5)